### PR TITLE
Fingerprint - 2020 Mazda CX-9

### PR DESCRIPTION
**Pull request by b801r**
Added values for 2020 Mazda CX-9 to selfdrive/car/mazda/values.py
Tested changes and system engages and works as expected.
I don't know how to exclude the changes on the interface.py file in the pull request.
Apologies if I did something incorrect, new to OpenPilot and new to GitHub.
Thanks!

### CODE REVIEW

**Comment by vanillagorillaa** [comment]
Be sure to provide the dongleID of the device that these versions were gathered from

**Comment by sshane** [comment]
any updates @b801r

**Comment by b801r** [comment]
DongleID: 139b81648a954ae7
Apologies for the delay, please let me know if any other info is required. Thanks!

File: selfdrive/car/mazda/interface.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     ret.safetyConfigs = [get_safety_config(car.CarParams.SafetyModel.mazda)]
     ret.radarUnavailable = True
 
-    ret.dashcamOnly = candidate not in (CAR.CX5_2022, CAR.CX9_2021)
+    ret.dashcamOnly = False
```
Comment (incognitojam) [inline selfdrive/car/mazda/interface.py:19]:
```suggestion
    ret.dashcamOnly = candidate not in (CAR.CX5_2022, CAR.CX9_2021)
```

**Comment by brendandebeasi** [comment]
Hello is there anything I can do to validate if a 2020 mazda cx9 would work with comma? Happy to test anything needed

**Pull request merged**